gbasis/parsers.py

```diff
@@ -99,15 +99,15 @@ def parse_gbs(gbs_basis_file):
     # pylint: disable=R0914
     with open(gbs_basis_file, "r") as basis_fh:
         gbs_basis = basis_fh.read()
-
+    # splits file into 'element', 'basis stuff', 'element',' basis stuff'
+    # e.g., ['H','stuff with exponents & coefficients\n', 'C', 'stuff with etc\n']
     data = re.split(r"\n\s*(\w[\w]?)\s+\w+\s*\n", gbs_basis)
     dict_angmom = {"s": 0, "p": 1, "d": 2, "f": 3, "g": 4, "h": 5, "i": 6, "k": 7}
     # remove first part
     if "\n" in data[0]:  # pragma: no branch
         data = data[1:]
-
-    atoms = data[::2]
-    basis = data[1::2]
+    atoms = data[::2]  # stride of 2 get the ['H','C', etc] from e.g., above
+    basis = data[1::2]  # starting with 'stuff' take strides of 2 to skip elements
```

Comment:
Could you merge these into a single comment above the line? We try to avoid inline comments